src/views/MessagePage.tsx

```diff
@@ -157,16 +163,28 @@ const MessagePageImpl: React.FC<RouteComponentProps & { fileStore: IFileStore }>
                   {segment.definition && segment.definition.description ? segment.definition.description : segment.name}
                 </ListSubheader>
                 {(segment.children as any[]).map((field, fieldIndex) => !field ? undefined : (
-                  <ListItem
-                    key={`parsedField-${segmentIndex}-${fieldIndex}`}
-                    className={selectedSegmentIndex === segmentIndex && selectedFieldIndex === fieldIndex ? classes.selectedField : classes.field}
-                    onClick={() => setSelected([segmentIndex, fieldIndex])}
-                    >
-                    <ListItemText
-                      primary={field.value ? field.value : '(empty)'}
-                      secondary={field.definition && field.definition.description ? field.definition.description : field.name}
-                    />
-                  </ListItem>
+                  <div>
+                    {field.children ? <div>
```

Comment:
When using ternaries inside JSX my preferred style is to use extra parens so that each section is clearly separated. I.e.

```jsx
trueOrFalse ? (
  <div>
    <h1>It was true!</h1>
    <h2>and not false.</h2>
  </div>
) : (
  <div>
    <h1>It was false!</h1>
    <h2>and not true.</h2>
  </div>
)
```